impress-org/givewp
# Refactor: Move Admin UI repository code into Controllers

Resolves #

## Description

This PR refactors the Donors, Donations, and DonationForms repositories and it moves the _request logic_ of each repository into the separate controller class. Also, the code is updated to use some of the PHP 7.0 features.

## Affects

 Donors, Donations, and DonationForms repositories

## Testing Instructions

These changes only affect the list tables of Donors, Donations, and Donation Forms pages, so to test this, it should be enough to simply visit these pages and check that everything is working correctly ( list table data is loaded, filters are working, etc. )



## Pre-review Checklist
